### PR TITLE
[Docs](docs) Modify plugin documents

### PR DESCRIPTION
**Pull request by yongjinhou**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Modify plugin documents，add description for importing slow query logs.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [x] Has document been added or modified
* [ ] Does it need to update dependencies
* [x] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by yongjinhou** [comment]
run buildall

**Comment by yongjinhou** [comment]
run p0

File: docs/zh-CN/docs/ecosystem/audit-plugin.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ auditloader plugin的配置位于`${DORIS}/fe_plugins/auditloader/src/main/assem
 
 打开 `plugin.conf` 进行配置。配置项说明参见注释。
 
+当前，审计日志插件支持将慢查询日志导入到单独的慢表 `doris_slow_log_tbl__` 中，Doris 中默认关闭，在审计日志的配置文件中，增加 `enable_slow_log = true`，开启该功能。并且可以在 FE 配置文件中修改 `qe_slow_log_ms` 项来修改慢查询阈值。
```
Comment (luozenglin) [inline docs/zh-CN/docs/ecosystem/audit-plugin.md:45]: Please add version tag for new features

**Comment by yongjinhou** [comment]
run buildall

**Comment by yongjinhou** [comment]
run p0

**Comment by yongjinhou** [comment]
run p0

**Review by luzhijing — APPROVED** [review]
LGTM

**Pull request merged**